html/elements/input/datetime-local.json

```diff
@@ -0,0 +1,58 @@
+{
+  "html": {
+    "elements": {
+      "input": {
+        "input-datetime-local": {
+          "__compat": {
```

Comment:
As [before](https://github.com/mdn/browser-compat-data/pull/2198#issuecomment-395250429), I think this needs:

    "description": "<code>type=\"datetime-local\"</code>"